features/shop/apply_correct_shipping_fee_with_taxes_on_order.feature

```diff
@@ -0,0 +1,39 @@
+@ui-cart
+Feature: Apply correct shipping fee with taxes on order
+    In order to pay proper amount for shipping
+    As a Customer
+    I want to have correct shipping fees and taxes applied to my order
+
+    Background:
+        Given the store is operating on a single channel
```

Comment:
For a separate PR, but we use present simple tense everywhere, so this should be:

```
Given the store operates on a single channel
```